hyperdash_cli/cli.py

```diff
@@ -411,6 +424,10 @@ def main():
     run_parser.add_argument('args', nargs=argparse.REMAINDER)
     run_parser.set_defaults(func=run)
 
+    pipe_parser = subparsers.add_parser('pipe')
+    pipe_parser.add_argument('--name', '-name', '--n', '-n', required=True)
```

Comment:
Food for thought, is there a way we could support autonaming? The key would be if we could extract the script name (or some meta data) from the pipe (otherwise it'd be impossible to know what model the pipe came from).

From what I've found pipe is purely stdin, with no meta data. In theory we could autoname by the first stdin line, but that seems brittle/collision-prone.